mtp_noms_ops/apps/security/views/check.py

```diff
@@ -113,6 +115,34 @@ def get_form_kwargs(self):
         )
         return form_kwargs
 
+    @staticmethod
+    def is_latest_state_active(auto_accept_rule):
+        return sorted(
+            auto_accept_rule['states'], key=lambda x: x['created']
+        )[-1]['active']
+
+    def is_existing_active_auto_accept_rule(self, api_session, debit_card_sender_details_id, prisoner_profile_id):
+        query_existing_auto_accept_rule = api_session.get(
+            '/security/checks/auto-accept?{}'.format(
+                urlencode((
```

Comment:
you don't need to url-encode these if you pass them in as a dict to the `params` argument

Reply:
Would that then URLencode them itself, or send the arguments as `body` parameters? I'm not sure I like the idea of a GET request with `body` parameters

Reply:
`data` is used for POST/body whereas `params` are always added as a query string